testmon/process_code.py

```diff
@@ -147,21 +152,17 @@ def gap_marks_until(lines, start, end):
 
 
 def covered_unused_statement(start, end, coverage):
-    while start <= end:
-        if start in coverage:
-            return True
-        start += 1
-    return False
+    return any(i in coverage for i in range(start, end))
 
 
-def create_fingerprints(afile, special_blocks, coverage):
+def create_fingerprints(lines, special_blocks, coverage):
     line_idx = 0
     result = []
-    while line_idx < len(afile):
+    while line_idx < len(lines):
         line_idx += 1
```

Comment:
Just change position 😉 
```suggestion
        line = lines[line_idx]
        line_idx += 1
```